src/types/types.ts

```diff
@@ -297,3 +297,8 @@ export type UserCredentials = Record<
   string,
   { expires: string; credentials: { clientId: string; accessToken: string } }
 >;
+
+export type TokenBearer = Record<
+  string,
+  { access_token: string; token_type: 'Bearer' }
+>;
```

Comment:
I think these 2 types are incorrect, they're not `Record<string, XXX>`, but directly `XXX`.
```js
export type UserCredentials = { expires: string; credentials: { clientId: string; accessToken: string } };

export type TokenBearer = { access_token: string; token_type: 'Bearer' };
```

A `Record` is like a Map (but using an object), also called a dictionary. This isn't what it is here.

Reply:
I just noticed that the Treeherder implementation stores the userCredentials in this way using the url.
```
userCredentials           "https://firefox-ci-tc.services.mozilla.com": {
                                       expires: ... 
                                       credentials: {
                                              clientId: ...
                                              accessToken: ...
                                       }
                           }
```

I believe that's why the Record for userCredentials. Should we still go with removing the Record for userCredentials?